gunicorn/workers/gaiohttp.py

```diff
@@ -4,14 +4,25 @@
 # See the NOTICE for more information.
 
 import sys
+import warnings
 
-if sys.version_info >= (3, 3):
+if sys.version_info >= (3, 4):
     try:
         import aiohttp  # NOQA
     except ImportError:
         raise RuntimeError("You need aiohttp installed to use this worker.")
     else:
-        from gunicorn.workers._gaiohttp import AiohttpWorker
-        __all__ = ['AiohttpWorker']
+        try:
+            from aiohttp.worker import WSGIServer  # NOQA
+            from aiohttp.worker import GunicornWebWorker as AiohttpWorker
+        except ImportError:
+            from gunicorn.workers._gaiohttp import AiohttpWorker
+
+            warnings.warn(
+                "AiohttpWorker is deprecated please install "
+                "aiohttp >= 1.2.0 ",
+                DeprecationWarning)
```

Comment:
I think we do use https://github.com/benoitc/gunicorn/blob/master/gunicorn/util.py#L550 for this kid of use cases. With, `util.warn`, people can see that the worker has been deprecated without passing `-Wd` or `-Wall` to `python`.